com.bitwig.BitwigStudio.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.bitwig.BitwigStudio</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <name>BitwigStudio</name>
+  <summary>BitwigStudio digital audio workstation</summary>
+  <description>
+    <p>Modern music production for Windows, macOS, and Linux.</p>
```

Comment:
Ideally the description should be expanded to replicate at least some of the content from the main website. It will make app page more attractive but ultimately it's up to you,

Reply:
We'll come back to it for the next release.

Reply:
Done.
I copied the new one over.